samples/csharp/MicrosoftGraphSDK/Assignment.cs

```diff
@@ -36,6 +36,31 @@ public class Assignment
             }
         }
 
+        /// <summary>
+        /// Get all the assignments from the class
+        /// </summary>
+        /// <param name="client"></param>
+        /// <param name="classId"></param>
+        /// <returns>EducationAssignment</returns>
+        public static async Task<IEducationClassAssignmentsCollectionPage> GetAssignments(
+            GraphServiceClient client,
+            string classId
+            )
+        {
+            try
+            {
+                return await client.Education
+                    .Classes[classId]
+                    .Assignments
+                    .Request()
+                    .GetAsync();
+            }
+            catch (Exception ex)
+            {
+                throw new GraphException($"GetAssignments call: {ex.Message}", classId);
+            }
+        }
```

Comment:
create the same method for MicrosoftGraphSDKBeta project

Reply:
Thanks for your comment, changes have been added to the latest commit